### PR TITLE
improvement: Improve genezio addClass output

### PR DESCRIPTION
**Pull request by bhavyastar**
# Pull Request Template

## Type of change

-   [ ] 🍕 New feature
-   [ ] 🐛 Bug Fix
-   [ ] 🔥 Breaking change
-   [x] 🧑‍💻 Improvement
-   [ ] 📝 Documentation Update

## Description

This PR improves Genezio addClass output

## Checklist

-   [x] My code follows the contributor guidelines of this project;
-   [ ] I have updated the documentation;
-   [ ] I have added tests;
-   [ ] New and existing unit tests pass locally with my changes;

### CODE REVIEW

**Comment by bhavyastar** [comment]
Please review this @andreia-oca

**Review by andreia-oca — CHANGES_REQUESTED** [review]

**Comment by bhavyastar** [comment]
Hey @andreia-oca 
Can you please review the changes?

**Review by andreia-oca — APPROVED** [review]
LGTM! Thank you for your contributions 🎉

File: src/commands/addClass.ts [hunk 1]
```diff
@@ -40,7 +40,9 @@ export async function addClassCommand(classPath: string, classType: string) {
     const classExtension = className.split(".").pop();
 
     if (!classExtension || className.split(".").length < 2) {
-        throw new UserError("Please provide a class name with a valid class extension.");
+        throw new UserError(
+            `Please provide a class name with a valid class extension. Currently supporting:${supportedExtensionsString}`,
```
Comment (andreia-oca) [inline src/commands/addClass.ts:44]:
```suggestion
            `Please provide a class name with a valid class extension. Currently supporting: ${supportedExtensions}`,
```

**Comment by andreia-oca** [comment]
This PR seems inactive - I will close it. If there are any changes to the branch, we can reopen it.

**Pull request closed without merging**